specs/core/oslc-core.html

```diff
@@ -594,18 +594,18 @@ <h2>Resource Constraints</h2>
               an LDPC for all the shapes for a spec version, at a URI defined by the following pattern:
             </p>
             <pre>
-      http://open-services.net/ns/[vocab short name]/shapes/[SPEC-version]
+      http://open-services.net/ns/[vocab-short-name]/shapes/[SPEC-version]
```

Comment:
Should be "http://open-services.net/ns/[vocab-short-name]/shapes/[SPEC-version]#" if that's a prefix declaration.

Reply:
Its not necessarily a prefix definition, its a URI pattern.

Reply:
I am just afraid someone will use that to form URIs without #

Reply:
That is the URI of an LDPC, so it shouldn't end in #. The examples below all show fragments of that URI. Now I'm not sure if that's why we used / before - the shape resource URI is an LDPC, not a document. Maybe we need to leave it a document and remove the reference to LDPC.